### PR TITLE
Enable 0-base using config.json; add relevant tests

### PR DESCRIPTION
**Pull request by RRathna**
Signed-off-by: Rathna Ramesh <rathna1993@gmail.com>

### CODE REVIEW

File: tests/myst/exec-package/Makefile [hunk 1]
```diff
@@ -5,6 +5,7 @@ APPDIR = appdir
 APPNAME = hello
 CFLAGS = -fPIC -g
 LDFLAGS = -Wl,-rpath=$(MUSL_LIB)
+LDFLAGS += -L$(LIBDIR) -lopenenclave -L$(BUILDDIR)/mbedtls/lib -lmbedcrypto
```
Comment (mingweishih) [inline tests/myst/exec-package/Makefile:8]: Do you really need this?

Comment (RRathna) [reply tests/myst/exec-package/Makefile]: Modified to only include -lopenenclave - needed by
extern const void* __oe_get_enclave_start_address(void);
extern const void* __oe_get_enclave_base_address(void);

**Review by mingweishih — APPROVED** [review]

**Comment by mingweishih** [comment]
@salsal97 pipeline seems not triggered/showed as part of checks. Can you check?

**Comment by salsal97** [comment]
@mingweishih, I see the pipeline run - is this resolved?

**Comment by RRathna** [comment]
> @mingweishih, I see the pipeline run - is this resolved?

Bo added me to the contributors list. I was able to start the pipeline with '/AzurePipelines run'. The issue is resolved.

**Pull request merged**